### PR TITLE
added 15112 :)

### PR DESCRIPTION
**Pull request by fredasu**
# Summary of Changes

- Please indicate the classes you are modifying:
  - made 15112

# Checklist:

Make sure that your contributions:

- [ ] Are accurate and will not mislead the reader (put in a disclaimer about being remote S21)
- [ ] Do not encourage or cause academic integrity violations
- [ ] Do not offend professors or unfairly criticize the university

### CODE REVIEW

File: cs_core/15112.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+- The first quiz, as mentioned above, is usually a bit rough.
+- HW gradually get harder over time, but the difficulty will flatten out by 
+week 4 according to the teachers. IMO I think the difficulty level depends on how
+much experience you've had with the topic before.
+- Technology failures: back up your code somewhere, and don't submit for the
+first time too close to the deadline.
+
+## Resources
+
+- syllabus if you want to look at the notes/schedule: https://www.cs.cmu.edu/~112/syllabus.html
```
Comment (mikinty) [inline cs_core/15112.md]: Do you mind formatting the link here to be markdown friendly? I think github by default recognizes it, but just to be more official 

```
[title](https://www.example.com)
```
```suggestion
- syllabus if you want to look at the notes/schedule: [https://www.cs.cmu.edu/~112/syllabus.html](https://www.cs.cmu.edu/~112/syllabus.html)
```

File: cs_core/15112.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+when people are still adjusting to the course, but it's definitely not a regular thing.
+- Term project counts for a good chunk of your grade so spend quality time on it.
+
+## What to watch out for
+
+- The first quiz, as mentioned above, is usually a bit rough.
+- HW gradually get harder over time, but the difficulty will flatten out by 
+week 4 according to the teachers. IMO I think the difficulty level depends on how
+much experience you've had with the topic before.
+- Technology failures: back up your code somewhere, and don't submit for the
```
Comment (mikinty) [inline cs_core/15112.md:101]: Is `git` a reasonable recommendation here for file management, or is that too complicated for 15-112? 

When I was at CMU, CS majors had to learn `git` in some class (although you barely did), but 112 is taken by so many non-CS majors, it's unlikely that most people will know what it is. 

Version control is extremely important at higher level classes, but getting a headstart in 112 could be a good idea

**Review by mikinty — CHANGES_REQUESTED** [review]
Can you please add the link to this on the main table of contents please?

Overall, fantastic article, as it gives good insight and personal experiences into what it is like to take the class. Great for people considering taking the class or people currently in it. Thanks!

**Comment by fredasu** [comment]
Hi, thanks for the feedback! Sorry for the late reply.

Is there an easy way to make changes or would I need to re-upload again?
I'm new to this github stuff.

Also, I don't think github was needed for 15-112 because they allowed other
forms of backup as well, such as Google Drive.

Thanks!

Freda

**Comment by fredasu** [comment]
Should I add more information about how to back things up regarding github?
Also, I created github with a personal account a while ago, but don't check
this email as frequently anymore since I have my school email,
***@***.*** so would you mind switching to that one instead?

Thank you!

**Comment by mikinty** [comment]
No worries about the delay.

Github sends an email based on your account preferences, so you can switch to your school email if you'd like. 

You should claim the [free GitHub pro and a bunch of other goodies](https://education.github.com/pack) with your student email.

To make changes, you can just add more commits here. I'd prefer it if you squash it so we don't have too many commits for this PR.

**Comment by mikinty** [comment]
> Hi, thanks for the feedback! Sorry for the late reply. Is there an easy way to make changes or would I need to re-upload again? I'm new to this github stuff. Also, I don't think github was needed for 15-112 because they allowed other forms of backup as well, such as Google Drive. Thanks! Freda

Yeah so using `git` is probably a more legit way of backing things up, esp for finding stuff in history, but I agree Google Drive is honestly fine for 15-112. 122 and beyond is when I would consider using `git`, and classes like `213` pretty much force you to do it.

**Review by mikinty — APPROVED** [review]
Thank you :)

**Pull request merged**